woocommerce/woocommerce-gutenberg-products-block
# Replace usage of `cart-checkout/button` with `@woocommerce/base-components/button`

Replaces usage of `cart-checkout/button` with `@woocommerce/base-components/button`, since the base component was added for use outside of the cart. They are identical.

Fixes #2735

### How to test the changes in this Pull Request:

1. Smoke test cart/checkout shows buttons correctly 
2. See Place order button on checkout 
3. See apply coupon button on cart
